tags/log.md

```diff
@@ -1,5 +1,9 @@
 ---
 aliases: ['sendlog', 'logs', '🪵']
+embed:
+  title: Upload Logs
+  color: orange
 ---
+The recommended site to upload your logs to is [mclo.gs](https://mclo.gs/).
 
-Please send logs: https://media.discordapp.net/attachments/923671549758820434/1027408644289077268/unknown.png
```

Comment:
we usually use this tag if we need logs from someone in the support channels. So we actually wanna keep this I think.